src/jrd/inf_pub.h

```diff
@@ -217,6 +217,7 @@ enum  info_db_implementations
 	isc_info_db_impl_linux_alpha = 83,
 	isc_info_db_impl_linux_arm64 = 84,
 	isc_info_db_impl_linux_ppc64el = 85,
+	isc_info_db_impl_linux_m68k = 86,
```

Comment:
Hi Mariuz!

This was for Firebird2.5 which did not have any support for ppc64 back then, but it was added to FB3.0.

I can backport the ppc64 changes to 2.5 if you want to and update this PR accordingly.

Adrian